src/omero_zarr/cli.py

```diff
@@ -82,8 +82,9 @@ def _configure(self, parser):
 
         parser.add_argument(
             "--bf",
-            action="store_true",
-            help="Use bioformats2raw to read from managed repo",
+            type=str,
+            help="Full path to bioformats2raw base directory."
+                 "Use bioformats2raw to read from managed repo.",
```

Comment:
or clientPath, right?

Reply:
True. But maybe something simpler like "; for using bioformats2raw to export the image." would be better anyway?